contracts/FlowIDTableStaking.cdc

```diff
@@ -1429,6 +1502,9 @@ pub contract FlowIDTableStaking {
     init(_ epochTokenPayout: UFix64, _ rewardCut: UFix64) {
         self.account.save(true, to: /storage/stakingEnabled)
 
+        let slotLimits: {UInt8: UInt8} = {1: 3, 2: 3, 3: 3, 4: 3, 5: 3}
```

Comment:
We should also select different defaults before merging this. Assuming we intend to invoke slot selection alongside the auto-recovery process, this will break epoch transitions on many transient networks we use currently (Localnet, Benchnet)

Reply:
Ok. Changing to UInt16 is easy enough. The defaults were selected to be friendly to my test and they are not rational. What do you think they should be @jordanschalm? I will update them and revise the test accordingly.